modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -108,7 +109,7 @@ def upload_supporting_documents
 
           documents.each do |document|
             claim_document = claim.supporting_documents.build
-            claim_document.set_file_data!(document, params[:doc_type], params[:description])
+            claim_document.set_file_data!(document, EVSS_DOCUMENT_TYPE, params[:description])
```

Comment:
why not still accept `doc_type` if provided but fall back to L023 if not provided?

Reply:
ah, is `doc_type` not in the documented schema...?

Reply:
^^^ Exactly

Reply:
I think this is fine for now, but I like the idea of adding an optional field and then switching to `EVSS_DOCUMENT_TYPE = params[:doc_type] || 'L023'` in the future